script.js

```diff
@@ -1,4 +1,144 @@
+// two different modes
+var diceGame = "diceroll";
+var ChooseOrder = "choose order";
+// start with diceroll game first
+var gameMode = diceGame;
+// who's turn is it?
+var currentPlayer = 1;
+// to keep track of the players' dice rolls
+var diceRollOfP1 = [];
+var diceRollOfP2 = [];
+// Players' Chosen Numbers
+var p1Num;
+var p2Num;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (gameMode == diceGame) {
+    var newDiceRoll = getNewDiceRoll();
+    // console.log(newDiceRoll, "runs first");
+    gameMode = ChooseOrder;
+    return (
+      "Hello Player " +
+      currentPlayer +
+      "! <br> You rolled " +
+      newDiceRoll[0] +
+      " on dice 1 and " +
+      newDiceRoll[1] +
+      " on dice 2. <br><br> Please choose your order by inputting the number 1 or the number 2."
+    );
+  }
+
```

Comment:
I realise you have several of these long return statements. You could create a helper function that does this for you. e.g. generateMessage(gameMode, currentPlayer). This function should return a custom message depending on the game mode and current player.